change_problem.py

```diff
@@ -26,13 +26,17 @@
 matplotlib.use("agg")
 import matplotlib.pyplot as plt
 
-# TODO: Enter your token here
-token = 'YOUR-TOKEN-HERE'
+def set_token():
```

Comment:
`set_token()` seems to imply that the user can actually "set" the token to parameters the user specifies. (e.g. `set_token(<parameters that edits token>)`. I think `get_token()` is what wanted here.

```
my_token = get_token()
```

Please see section 1 on getters and setters: https://www.datacamp.com/community/tutorials/property-getters-setters